ndcube/ndcube.py

```diff
@@ -6,13 +6,14 @@
 import astropy.nddata
 import astropy.units as u
 import numpy as np
-import sunpy.coordinates
-from astropy.wcs.wcsapi import BaseLowLevelWCS
+import sunpy.coordinates  # pylint: disable=unused-import  # NOQA
```

Comment:
This import isn't used.  And if it was it would make `ndcube` dependent on `sunpy` again.  Aren't we trying to avoid that?

Reply:
I think we should probably keep this, maybe with a try block when we actually remove the sunpy dependency (which isn't done yet).

Reply:
Sounds good.

Reply:
This still needs a ```try...except...``` on the sunpy import?